README.adoc

```diff
@@ -278,33 +277,28 @@ file:
 include::finish/src/test/java/it/io/openliberty/guides/beanvalidation/BeanValidationTest.java[tags=**;!copyright]
 ----
 
-The `@BeforeClass` annotation is placed on a method that runs before any of the test cases. In this
-case, the `oneTimeSetup` method retrieves the port number for the Open Liberty server and builds a
-base URL string that is used throughout the tests.
-
-See the following descriptions of the test cases:
+Place the `@BeforeClass` annotation on a method that runs before any of the test cases. The `oneTimeSetup` method retrieves the port number for the Open Liberty server and builds a
+base URL string that is used throughout the tests, which are described as follows:
 
-* `testNoFieldLevelConstraintViolations` verifies that no constraint violations occur when data is
-supplied to the `Astronaut` and `Spacecraft` beans' attributes that passes the defined constraints.
+* The `testNoFieldLevelConstraintViolations` test case verifies that no constraint violations occur when data is
```

Comment:
Maybe instead of the "that passes the defined constraints" just add valid before data like: "... test case verifies that no constraint violations occur when valid data is..."

Reply:
Yes, adding "valid" is good. Maybe we should also remove "no constraint violations occur" and instead say, "constraint violations do not occur"...it sounds a bit more natural, I think.